File: Javascript/2751-2800/2798. Number of Employees Who Met the Target.js
// https://leetcode.com/problems/number-of-employees-who-met-the-target/

var numberOfEmployeesWhoMetTarget = function(hours, target) {
    let empCount = 0;

    for(let empHours of hours)
        if(empHours >= target)
            empCount ++;

    return empCount;
};

File: Python/2751-2800/2798. Number of Employees Who Met the Target.py
# https://leetcode.com/problems/number-of-employees-who-met-the-target/

class Solution:
    def numberOfEmployeesWhoMetTarget(self, hours: List[int], target: int) -> int:
        empCount = 0

        for empHours in hours:
            if empHours >= target:
                empCount += 1

        return empCount
